tests/integration/models/schedule_model/modify_model.rb

```diff
@@ -0,0 +1,62 @@
+require 'openstudio'
+
+# puts OpenStudio::Model::Model.methods.filter { |m| m.to_s.upcase.include? "LOAD"}
+# exit
+model = OpenStudio::Model::Model.load("small_office.osm").get
```

Comment:
Is it a design choice to make the model in code vs including and .os (maybe to keep OS version consistent), or am I missing something?